ergoCubSN000/calibrators/left_arm-calib.xml

```diff
@@ -30,7 +30,7 @@
 	</group>
 
 	<!-- motor's encoder of joint 7 can't read -->
-	<param name="CALIB_ORDER"> (4 5 6) (2) (0) (1) (3) (7 8 9 10) (11 12) </param> 
+	<param name="CALIB_ORDER"> (4 5 6) (3) (2) (0) (1) (7 8 9 10) (11 12) </param> 
```

Comment:
Something we need to review.
cc @Nicogene @sgiraz @davidetome @AntonioConsilvio @mfussi66.

Reply:
Seems ok to me, what is the reason of calibrating first the joint 3 ?

Reply:
I think this modification was done by @AntonioConsilvio, but is also in the branch upstream already. That's why as I mentioned in https://github.com/robotology/robots-configuration/pull/458#issuecomment-1472414537 I need to review first those modifications before the PR is ready for review

Reply:
Thanks! Conversation resolved!